client/src/components/CreateSession.jsx

```diff
@@ -12,12 +12,16 @@ class CreateSession extends React.Component {
     this.state = {
       searchQuery: '',
```

Comment:
I'm getting an error in the console about the input form not always being a controlled component. It looks like you set it up as one and it doesn't seem to affect anything on the page, so I think it's a minor bug.

Reply:
Strange! I'm not able to reproduce locally - how did you see this? Did you pull down this branch and see how it worked on your computer? I'm seeing a console warning about input elements requiring autocomplete attributes but it looks like it's on Tom's login screen